src/lib/redux/siteData.ts

```diff
@@ -25,4 +25,8 @@ const setCurrentSite = (newSite: any): void => {
   store.dispatch(setCurrSite(newSite));
 };
 
-export { refreshSiteData, setCurrentSite };
+const addCustomer = (customer: any): void => {
+  store.dispatch(addNewCustomer(customer));
```

Comment:
I'm a little iffy on my naming -- I was looking at the example of `setCurrSite` and `setCurrentSite` and assumed it'd be good to use separate names, so I have `addCustomer` and `addNewCustomer`. But we also do `createCustomer`, so I'm not sure if this naming is more confusing / if I should be more explicit that it's to redux??

Reply:
Renamed this one but leaving the other one as `addCustomer` just because it will yell at me if I try 'Import declaration conflicts with local declaration'